docs/defi/chain-key-tokens/ckbtc/overview.mdx

```diff
@@ -71,14 +74,22 @@ Learn more in the [Bitcoin integration technical overview](/docs/references/bitc
 
 - [Chain-key tokens](https://learn.internetcomputer.org/hc/en-us/articles/34211397080980-Chain-Key-Tokens).
 
-- [ckBTC](https://github.com/dfinity/ic-js/tree/main/packages/ckbtc) JavaScript library.
-
 - [GitHub repository](https://github.com/dfinity/ic/tree/master/rs/bitcoin/ckbtc/minter).
 
-- [Local development workflow](/docs/build-on-btc/using-regtest).
+- [ckBTC overview](https://dashboard.internetcomputer.org/bitcoin) on the ICP Dashboard.
+
+- [ckTESTBTC overview](https://dashboard.internetcomputer.org/testbtc) on the ICP Dashboard.
+
+- [ckbtc](https://github.com/dfinity/ic-js/tree/main/packages/ckbtc) JavaScript library for interfacing with the ckBTC minter.
+
+- [ledger-icrc](https://github.com/dfinity/ic-js/tree/main/packages/ledger-icrc) JavaScript library for interfacing with the ckBTC ledger.
+
+- [icrc-ledger-agent](https://crates.io/crates/icrc-ledger-agent)
```

Comment:
we should probably also add
https://crates.io/crates/icrc-ledger-types
https://crates.io/crates/icrc-ledger-client
```suggestion
- [icrc-ledger-agent](https://crates.io/crates/icrc-ledger-agent) Agent for interacting with ICRC-1 ledgers.
```

Reply:
I intentionally did not include them. I could add in the description that you added that this agent uses the `icrc-ledger-types` crates 🤔 

can you shortly enlighten me how `icrc-ledger-client` fits into all of this? when would I use this?

Reply:
isn't the intro before the libraries enough? 🤔 

> The following libraries simplify creating actors for the ckBTC ledger:

note that I am talking about creating actors with this library here. it might become confusing to the reader if I add the suggested info behind the libs.

Reply:
I added the libraries, I hope it is fine now